i18n/translation-en.json

```diff
@@ -267,7 +267,8 @@
     },
     "shared": {
       "page": "Page",
-      "precision": "Precision"
+      "precision": "Precision",
+      "enableSnapping": "Enable snapping for measurement tools"
```

Comment:
Let me know if the label should be changed.
didn't want to say "all measurement tools" because ellipse doesn't have snap mode